### DISCUSSION TITLE
Welcome to hass-amber-electric Discussions!

### DISCUSSION THREAD

**Discussion by lewisbenge** [lewisbenge/hass-amber-electric#28] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by micarex** [comment]
Thanks for setting this up @lewisbenge.   

Hi everyone, I'm a renewable energy enthusiast and I've been using Amber APIs for automation of my PV/battery system and relay control using various techniques to switch on/off household loads (hot water heating, aircon, and various devices plugged into GPOs).  I do most of the automation using Home Assistant but also Fronius inverter inbuilt capability, and a few other devices such as Ikea Tradfri wireless GPOs and Netio wired GPOs.
